### DISCUSSION TITLE
Continuous 'VerboseMoveStats' ?

### DISCUSSION THREAD

**Discussion by peteschaefer** [LeelaChessZero/lc0#2096] [Q&A] [answered]
I understand that the UCI option 'VerboseMoveStats' prints additional info about Leela's search variables (Q, N, P, etc.). 
Great.
But this info is only printed at the end of a search.

When 'go infinite' there is no way to get those stats regularly. 

Nibbler obviously can do it, but I don't know how. (maybe by calling "stop - go infinite" in regular intervals, but that seems clunky to me).

Does anybody have a clue on how to achieve this ?

**Comment by mooskagh** [comment]
You can get N and Q (or rather WDL) by setting high multipv and `--per-pv-counters`, it will go through the normal UCI messages.

P is the policy value and doesn't change with time, what Nibbler does is `go nodes 1` first an then `go infinite`.

**Comment by peteschaefer** [comment, marked as answer]
Thanks.

I solved it with options "UCI_WDL", "VerboseMoveStats" and "LogLiveStats".

LogLiveStats is a "hidden" option that you only see with --help --show-hidden. 
A bit hard to find but it does what I want ;)
